lib/util/addExtraParamsTo.js

```diff
@@ -10,7 +10,11 @@ function addExtraParamsTo(routeArguments) {
     //  is null. WWARNER 08/21/2014
     paramsAndRequest.pop();
 
-    var extraParams = Array.prototype.slice.call(arguments, 1);
```

Comment:
was this actually leaking arguments? `Array.prototype.slice` creates a shallow copy